modules/queue/setting.go

```diff
@@ -37,8 +79,19 @@ func getQueueSettings(name string) (setting.QueueSettings, []byte) {
 }
```

Comment:
Maybe change `getQueueSettings` to use `QueueName` type to enforce type safety?

Reply:
That would make modules/setting dependent on modules/queue thus introducing a cyclic dependency.